### PR TITLE
chore: update managedkafka

### PR DESCRIPTION
**Pull request by JoeWang1127**
In #11308, library config of managedkafka is changed.

Update the library using
```
docker run \
 --rm \
 -u "$(id -u):$(id -g)" \
 -v "$(pwd):${workspace_name}" \
 -v "$(pwd)/../googleapis:${workspace_name}/googleapis" \
 -e GENERATOR_VERSION=2.50.0 \
 gcr.io/cloud-devrel-public-resources/java-library-generation:2.50.0 \
 --generation-config-path="${workspace_name}/generation_config.yaml" \
 --api-definitions-path="${workspace_name}/googleapis" \
 --library-names=managedkafka
```

### CODE REVIEW

File: java-managedkafka/README.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ See the [Authentication][authentication] section in the base directory's README.
 
 ## Authorization
 
-The client application making API calls must be granted [authorization scopes][auth-scopes] required for the desired Managed Service for Apache Kafka API APIs, and the authenticated principal must have the [IAM role(s)][predefined-iam-roles] required to access GCP resources using the Managed Service for Apache Kafka API calls.
+The client application making API calls must be granted [authorization scopes][auth-scopes] required for the desired Managed Service for Apache Kafka API APIs, and the authenticated principal must have the [IAM role(s)][predefined-iam-roles] required to access GCP resources using the Managed Service for Apache Kafka API API calls.
```
Comment (suztomo) [inline java-managedkafka/README.md:70]: This "API API" tells me that the YAML source shouldn't have "API" in it.

**Review by suztomo — CHANGES_REQUESTED** [review]
Can you remove "API" from the source of truth line in the YAML?

File: java-managedkafka/README.md (hunk 1 above)
Comment (suztomo) [reply java-managedkafka/README.md:70]: I'm asking to fix name pretty value. Not template.

Comment (JoeWang1127) [reply java-managedkafka/README.md]: Done.

**Review by suztomo — APPROVED** [review]
Thank you.

**Pull request merged**